lib/index.ts

```diff
@@ -23,16 +26,62 @@ function inspect(root: string, targetFile?: string, options?: any) {
         dockerImageId: result[1].imageId,
       };
       const pkg: any = result[1].package;
+      const dockerfileAnalysis = result[2];
+      const dockerfilePackages = dockerfileAnalysis
+        ? getDockerfileDependencies(dockerfileAnalysis.dockerfilePackages,
+                                    pkg.dependencies)
+        : [];
+
       pkg.docker = pkg.docker || {};
-      pkg.docker.baseImage = result[2];
       pkg.docker.binaries = result[1].binaries;
+      pkg.docker = {
+        ...pkg.docker,
+        ...dockerfileAnalysis,
+        dockerfilePackages,
+      };
+
       return {
         plugin: metadata,
         package: pkg,
       };
     });
 }
 
+// Iterate over the dependencies list; if one is introduced by the dockerfile,
+// flatten it's dependencies and append them to the list of dockerfile
```

Comment:
sorry for nitpicking but `it's` should be `its` :)

Reply:
Fixed